pylint_odoo/checkers/modules_odoo.py

```diff
@@ -32,6 +32,11 @@
         'xml-syntax-error',
         settings.DESC_DFLT
     ),
+    'E%d03' % settings.BASE_OMODULE_ID: (
+        'Detected test folder imported in module %s',
```

Comment:
Remove "detected", all lints are intended to detect issues

Reply:
Done